### PR TITLE
Report/chart

### PR DESCRIPTION
**Pull request by hd-chuong**
Generate sales report at "report.php".

- [ ] Generate sales record table

- [ ] Generate relevant chart

### CODE REVIEW

File: src/report.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        <div class="row">
+            <div class="col-sm">
+                <label for="recorddatestart">Start Date </label><input class="form-control mr-sm-2" type="date" id="recorddatestart" name="recorddatestart" value="2020-09-04" required>
+            </div>
+
+            <div class="col">
+            <label for="recorddateend">End Date </label><input class="form-control mr-sm-2" type="date" id="recorddateend" name="recorddateend" value="2020-10-16" required>
+            </div>
+            <div class="col">
+                <input type="radio" class="form-check-input" id="category" name="WHICHDATA" value="CATEGORY"><label for="category">Catergory </label><input class="form-control mr-sm-2" type="text" id="CATEGORYID" placeholder="Category Name"></p>
```
Comment (nickmcm92) [inline src/report.php]: Rename to category ID/implement category ID lookup

File: src/report.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            </div>
+
+        <div class="col">
+            <input type="radio" class="form-check-input" id="item" name="WHICHDATA" value="ITEM" checked="checked"><label for="item">Item </label><input class="form-control mr-sm-2" type="text" id="ITEMID" placeholder="Item Name">
```
Comment (nickmcm92) [inline src/report.php]: Rename to item ID/implement item ID lookup

**Review by nickmcm92 — CHANGES_REQUESTED** [review]

File: src/report.php (hunk 2 above)
Comment (hd-chuong) [reply src/report.php]: Just added more clarification on the label + the placeholder.

File: src/report.php (hunk 1 above)
Comment (hd-chuong) [reply src/report.php]: Just added more clarification on the label + the placeholder.

**Review by nickmcm92 — APPROVED** [review]

**Pull request merged**